xml/x509_issuers.xml

```diff
@@ -135,7 +204,8 @@
     <description>VMware ESXi w/Installer</description>
     <example>O=VMware Installer</example>
     <param pos="0" name="os.vendor" value="VMware"/>
-    <param pos="0" name="os.product" value="ESXi"/>
+    <param pos="0" name="os.family" value="VMware ESX/ESXi"/>
+    <param pos="0" name="os.product" value="VMware ESXi Server"/>
```

Comment:
Any reason there is a vendor in the family/product field here? We don't put Cisco into the ASA variant above this.

Reply:
I only used that value to be consistent with the existing values in the NTP database. My preference would be
`os.family` = `vSphere`
`os.product` = `ESXi`

Reply:
`os.family` = `VMware ESX/ESXi`
`os.product` = `VMware ESXi Server`
these match Nexpose expectations so if you plan to change them let us know as we would need to introduce some mappings